sync-core/src/main/java/com/cloudant/sync/query/IndexManager.java

```diff
@@ -94,18 +93,32 @@ public IndexManager(Datastore datastore) {
         queue = Executors.newSingleThreadExecutor();
 
         final String filename = datastore.extensionDataFolder(EXTENSION_NAME) + File.separator
-                                                                        + "indexes.sqlite";
+                                                                              + "indexes.sqlite";
         SQLDatabase sqlDatabase = null;
         try {
             sqlDatabase = queue.submit(new Callable<SQLDatabase>() {
```

Comment:
@rhyshort I had no plans of refactoring the IndexManager.  When did using SQLDatabaseQueue to interact with the database become a requirement?